mmv1/templates/terraform/examples/external_ssl_proxy_lb_mig_backend_custom_header.tf.erb

```diff
@@ -0,0 +1,188 @@
+# External SSL proxy load balancer with managed instance group backend
+
+# [START cloudloadbalancing_ext_ssl_proxy_lb]
+# VPC
+resource "google_compute_network" "default" {
+  name                    = "<%= ctx[:vars]['ssl_proxy_xlb_network'] %>"
+  provider                = google
```

Comment:
Because this is `min_version: beta` you'll need to specify `google-beta` as the provider on all resources instead of `google`

Reply:
@rileykarson  Thank you

Reply:
Note, _every_ `google_` resource will need that specified!

Reply:
Hi @rileykarson - Thank you for your patience. I have update my ssl example, to use only `google` provider as this example does not need any `google-beta`. Please advice.